### DISCUSSION TITLE
依赖节点切换，只有到US节点速度才正常，其余都不太行

### DISCUSSION THREAD

**Discussion by toolflow-labs** [getlantern/lantern-forum-cn#936] [General]
现在打开蓝灯（专业版），好像抽奖，每次轮到SG或者JP、KR节点，都巨慢，Google完全打不开。

**Comment by MaxMKMax** [comment]
@hustxujinkang 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by LuLu120325** [reply]
请帮我处理退款，给个明确的回复，谢谢，邮箱：2647611833@qq.com

**Reply by MaxMKMax** [reply]
@LuLu120325 您好. 请您提供我支付单据, 这样我才能帮您提交退款处理. 

请问您为什么需要退款呢? 要不您告述我们问题, 看我们能帮您解决吗? 然后就不需要退款.

**Comment by MaxMKMax** [comment]
@hustxujinkang 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。我们需要您的票号/LOG才能找出问题的根源.

**Comment by MaxMKMax** [comment]
@hustxujinkang 您可以帮助我们在应用程序中提交报告您的问题吗？然后给我票号. Thanks!
